pkg/test/runner/util.go

```diff
@@ -15,69 +15,74 @@
 package runner
 
 import (
+	"bytes"
 	"fmt"
 	"io/ioutil"
 	"os"
 	"os/exec"
 	"path/filepath"
 )
 
-func runCommand(pwd, name string, arg []string) (string, error) {
+func runCommand(pwd, name string, arg []string) (string, string, error) {
 	cmd := exec.Command(name, arg...)
 	cmd.Dir = pwd
-	o, err := cmd.CombinedOutput()
-	return string(o), err
+	var out bytes.Buffer
+	var stderr bytes.Buffer
+	cmd.Stdout = &out
+	cmd.Stderr = &stderr
+	err := cmd.Run()
+	return out.String(), stderr.String(), err
 }
 
 func copyDir(src, dst string) error {
-	_, err := runCommand("", "cp", []string{"-r", src, dst})
+	_, _, err := runCommand("", "cp", []string{"-r", src, dst})
 	return err
 }
 
 func gitInit(d string) error {
-	o, err := runCommand(d, "git", []string{"init"})
+	o, s, err := runCommand(d, "git", []string{"init"})
```

Comment:
Maybe stdout and stderr here are clearer.

Reply:
+1

Reply:
done